configs/processors/github.yaml

```
@@ -0,0 +1,115 @@
+# ArangoDB Configuration (embeddings and structures only)
+arangodb:
+  host: ${ARANGODB_HOST:localhost}
+  port: 8529
+  username: root
+  database: academy_store
+  # password: Set via environment variable ARANGO_PASSWORD
+
+processor:
+  # Embedding configuration - see embedder.yaml for details
+  embedder_config: ../embedder.yaml
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)